test/functionalTest/cases/0725_multiservice_ignore/service_ignore_without_multiservice.test

```diff
@@ -101,10 +102,11 @@ Date: REGEX(.*)
 =====================================================
 1
 0
+ftest
 --TEARDOWN--
```

Comment:
There was a bug related with this due to the subscriptoin cache that has been now fixed (https://github.com/telefonicaid/fiware-orion/issues/1186). Maybe this was related and it is no longer needed as the bug has been solved?